spd_db_mysql.cc

```diff
@@ -9658,6 +9708,7 @@ int spider_mysql_handler::append_from(
     str->q_append(SPIDER_SQL_FROM_STR, SPIDER_SQL_FROM_LEN);
     table_name_pos = str->length();
     append_table_name_with_adjusting(str, link_idx, sql_type);
+    append_index_hint(str, link_idx, sql_type);
```

Comment:
This is danger when index name is different between Spider table and data table. So I'll add a parameter after merging this changes for switching this operation. And also you can use "idx000" table parameter for adding index hints.